packages/pluggableWidgets/maps-web/CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ All notable changes to this widget will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 2.0.2 - 2021-07-12
+### Fixed
+- We have fixed the dynamic marker data source consistency check for MX 9.2 and above.
```

Comment:
Does this really impacts in the user experience? (Studio Pro)

Reply:
Yes, the datasource consistency check for maps is broken for mx 9.2 and above, because it assumes there is an object.